### PR TITLE
Add api tests for tags and account_data 

### PR DESCRIPTION
**Pull request by pik**
@Half-Shot I added `responses` as a dep but if you have a preferred library / configuration for mocking requests in tests I will be happy to switch to that.

### CODE REVIEW

**Comment by Half-Shot** [comment]
ok to test

File: test/api_test.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+class TestTagsApi:
+    cli = client.MatrixClient("http://example.com")
+    user_id = "@user:matrix.org"
+    room_id = "#foo:matrix.org"
+
+    @responses.activate
+    def test_get_user_tags(self):
+
```
Comment (Half-Shot) [inline test/api_test.py]: Mystery newline :fearful:

**Comment by Half-Shot** [comment]
I've shamefully only got around to testing this now. Looks good except for one sneaky newline.

**Comment by pik** [comment]
@Half-Shot apologies for neglecting this - but looking at it I'm really not sure why the build is failing :o 

It looks like the `responses` module which I added to mock requests is missing from the test environment (http://matrix.org/jenkins/job/PythonSDK-PR/96/console):

```
==================================== ERRORS ====================================
______________________ ERROR collecting test/api_test.py _______________________
test/api_test.py:1: in <module>
    import responses
E   ImportError: No module named responses
====================== 8 passed, 1 error in 0.21 seconds =======================
ERROR: InvocationError: '/bin/bash -c py.test test'
```
But it's present in setup.py (https://github.com/matrix-org/matrix-python-sdk/pull/63/files#diff-2eeaed663bd0d25b7e608891384b7298R50) so I don't know why that's not being sourced (or if travis doesn't run setup.py for some reason).

**Comment by Half-Shot** [comment]
I tested it on my system and it worked fine. I'll poke @erikjohnston

**Comment by erikjohnston** [comment]
@matrixbot retest this please

**Comment by erikjohnston** [comment]
@matrixbot retest this please

**Comment by erikjohnston** [comment]
@matrixbot retest this please

**Comment by erikjohnston** [comment]
Right, that all works now. Looks like the failure is a legitimate flake8 violation: http://matrix.org/jenkins/job/PythonSDK-PR/110/violations/

**Comment by Half-Shot** [comment]
Thanks. Well, I can merge this one.

**Pull request merged**